orchestra/frontend/config-overrides.js

```diff
@@ -0,0 +1,10 @@
+const path = require('path');
+const { override, babelInclude } = require('customize-cra');
+
+module.exports = override(
+  babelInclude([
+    path.resolve('node_modules/@b12/metronome'),
```

Comment:
i know we needed this hack to get metronome imports to work as it doesn't export a library properly. is there a less hacky solution?

Reply:
I remember the reason was because we don't compile the metronome library, and hence when using create-react-app, it doesn't understand the files as its built in webpack/babel solution doesn't compile files in the `/node_modules/` folder. I can't think of any alternative solutions other than fixing up metronome to compile and export the compiled package? What do you think?

Reply:
I agree. Since this isn't a quick fix to metronome, want to add a task to follow up on it later?